clayui.com/content/blog/2019-10-24-introducing-clay-v3.md

```diff
@@ -0,0 +1,135 @@
+
+</div>
+
+### 3.x Roadmap
+
+We are working on developer experience enhancements and delivering new features. We have a few things for our roadmap.
+
+#### Data Display
+
+To help teams create complex and extensible data displays, Clay offers different Table components that can be used and configured on a high level. However, we are collecting current and future use cases so that we can create documentation and examples to help developers effectively display data across different applications. As we discover common patterns and uses, we will further evaluate the best way to provide components for users.
+
+#### Data Visualization
+
+The proposal here is to analyze the problems we had with ClayCharts, many teams complained about its use, API, and inconsistencies with Lexicon. The goal is to listen to current users who worked with Charts to gather feedback and evaluate the best direction forward, whether that be a new charts library or adding API enhancements to our current implementation.
+
+ClayCharts should allow for both simple use cases and also allow customization for further complexity.
+
+An important point is performance, many onscreen charts should not be a problem, We will study the cases and help solve the performance issues.
+
+The criteria to get there:
+
+<div class="clay-ullist clay-ullist-margin-sm">
+
+- Performance
+- Flexibility
+- Consistency with Lexicon
+- Readable API
+
+</div>
+
+We are already working on [collecting feedbacks and you can leave your 2cents](https://github.com/liferay/clay/issues/2577).
+
+#### Layout and ClayManager components
+
+Still under research, we will be working to understand common component use cases that can become components in Clay that help develop Layouts more easily.
+
+Since we have several contexts for different components it can be a bit annoying to add many nested contexts to your project's root application. We will work in an aggregating context that will bring all our contexts together in one place, making it easier for teams that use Clay as their base and not just some components.
+
+#### A11y component
+
+A written component on top of react-axe that gives direct accessibility feedback on screen and only in the development environment. Inspired by the work for the new Facebook.
+
+This can help reinforce our work in following accessibility guidelines and help product teams follow.
+
+### Acknowledgments
+
+We are very excited about this delivery and are curious as to how people will use Clay going forward, we want to thank everyone who used Clay v2 and struggled to contribute, creating issues and sending PRs. Many thanks to all the comments we received about using Clay v2 that helped us to model this new version. Keeping this feedback is very important so we can keep evolving our implementation and we are available to hear and understand your use cases. We hope adoption continues to grow.
```

Comment:
Just a few suggestions.

```suggestion
We are very excited about this release and are looking forward to see how people use Clay going forward, we want to thank everyone who used Clay v2 and who contributed by creating issues and sending PRs. Many thanks to all the comments we received about using Clay v2 that helped us to model this new version. Keeping this feedback is very important so that we can keep evolving and creating a great experience for end users and developers. We look forward to continue growth.
```